src/olympia/reviewers/tests/test_views.py

```diff
@@ -8166,6 +8189,12 @@ def test_results(self):
             'Unlisted versions (2)',
             reverse('reviewers.review', args=['unlisted', addon.slug])
         ))
+        # mixed, only unlisted flagged
+        addon = self.expected_addons[2]
+        expected.append((
+            'Unlisted versions (1)',
+            reverse('reviewers.review', args=['unlisted', addon.slug])
+        ))
```

Comment:
Do we have any that have both unlisted and listed versions flagged to test that case?

Reply:
Hm, no. I can add one, that will increase the number of queries by 2 again. Is that ok?

Reply:
the number of queries in the test doesn't concern me, it's what the number of queries will be on prod for each view of the page.

Reply:
I agree. Although this patch doesn't add to the impact, it's already on prod today.

Reply:
Filed https://github.com/mozilla/addons-server/issues/15269